### PR TITLE
new: usr: TARGET_DATE defaults to yesterday.

### LINKED ISSUE #4: Confirm HALO_API_HOSTNAME is used for non-MTG API
Related to:
https://github.com/cloudpassage/cortex/issues/46

### PR DESCRIPTION
**Pull request by ashmastaflash**
Also supports HALO_API_HOSTNAME

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ S3 bucket you need to drop the events into.
 * HALO_API_KEY: sometimes referred to as Key ID
 * HALO_API_SECRET_KEY
 * HALO_API_HOSTNAME (optional, defaults to `api.cloudpassage.com`)
-* TARGET_DATE: Formatted like this: "2016-12-01"
+* TARGET_DATE: Formatted like this: "2016-12-01" (optional, defaults to yesterday)
```
Comment (mong2) [inline README.md]: should specify using iso 8601 format

Comment (ashmastaflash) [reply README.md]: Fixed.

File: tool/eventslib/utility.py [hunk 2]
```diff
@@ -1,8 +1,24 @@
+import datetime
 import os
 import re
 
 
 class Utility(object):
+    @classmethod
+    def iso8601_yesterday(cls):
```
Comment (eatwithforks) [inline tool/eventslib/utility.py]: can pull this function out and just use iso8601_arbitrary_days_ago and pass in 1. 
kind of redundant to use iso8601_yesterday.

Comment (ashmastaflash) [reply tool/eventslib/utility.py]: done

**Pull request merged**